synctl/cli.py

```diff
@@ -4214,7 +4310,7 @@ def create_command_options(self):
 
     def get_command_options(self):
         self.parser_get.add_argument(
-            'op_type', choices=['location', 'lo', 'test', 'application', 'app', 'cred', 'alert', 'alert-channel', 'result', 'pop-size'],
+            'op_type', choices=['location', 'lo', 'test', 'application', 'app', 'cred', 'alert', 'alert-channel', 'result', 'pop-size', 'pop-cost'],
```

Comment:
To make these two command easy to use, we can use abbreviations for them, the abbreviations for `pop-size` is `size` and the abbreviations for `pop-cost` is `cost`.
```suggestion
            'op_type', choices=['location', 'lo', 'test', 'application', 'app', 'cred', 'alert', 'alert-channel', 'result', 'pop-size', 'size', 'pop-cost', 'cost'],
```